### PR TITLE
Template admin

### PR DESCRIPTION
**Pull request by LeCaoDat**
image

### CODE REVIEW

**Comment by LeCaoDat** [comment]
ready

File: app/views/admin/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <span class="sr-only"><%= t ".toggle_navigation" %></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    </button>
+  </div>
+  <ul class="nav navbar-top-links navbar-right">
+    <li class="dropdown">
+      <%= link_to "", class: "dropdown-toggle", "data-toggle": "dropdown" do %>
+      <i class="fa fa-user fa-fw"></i> <i class="fa fa-caret-down"></i>
```
Comment (tranducquoc) [inline app/views/admin/layouts/_navbar.html.erb]: indent

File: app/views/admin/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      <% end %>
+      <ul class="dropdown-menu dropdown-messages">
+        <li>
+          <%= link_to root_path do %>
+          <i class="fa fa-home fa-fw"></i>
```
Comment (tranducquoc) [inline app/views/admin/layouts/_navbar.html.erb]: indent

File: app/views/admin/static_pages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            <div class="col-xs-3">
+              <i class="fa fa-user fa-5x"></i>
+            </div>
+            <div class="col-xs-9 text-right">
+              <div class="huge"><%= t "admin.layouts.manage_users" %></div>
+            </div>
+          </div>
+        </div>
+        <%= link_to "" do %>
+            <div class="panel-footer">
```
Comment (tranducquoc) [inline app/views/admin/static_pages/index.html.erb]: indent

File: app/views/admin/static_pages/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            <div class="col-xs-3">
+              <i class="fa fa-support fa-5x"></i>
+            </div>
+            <div class="col-xs-9 text-right">
+              <div class="huge"><%= t "admin.layouts.manage_categories" %></div>
+            </div>
+          </div>
+        </div>
+        <%= link_to "" do %>
+            <div class="panel-footer">
```
Comment (tranducquoc) [inline app/views/admin/static_pages/index.html.erb]: indent

File: app/views/admin/static_pages/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            <div class="col-xs-3">
+              <i class="fa fa-tasks fa-5x"></i>
+            </div>
+            <div class="col-xs-9 text-right">
+              <div class="huge"><%= t "admin.layouts.manage_products" %></div>
+            </div>
+          </div>
+        </div>
+        <%= link_to "" do %>
+            <div class="panel-footer">
```
Comment (tranducquoc) [inline app/views/admin/static_pages/index.html.erb]: indent

**Comment by tranducquoc** [comment]
commented

**Comment by LeCaoDat** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**